app/models/room.rb

```diff
@@ -28,6 +28,8 @@ class Room < ApplicationRecord
   has_many :gizmos, dependent: :destroy, inverse_of: :room, class_name: :Furniture
   accepts_nested_attributes_for :gizmos
 
+  validates :description, length: {maximum: 160}
```

Comment:
Some quick searches suggest that a reasonable estimate of the average character length for an English sentence is about 100, so I think we should allow at least 200 characters here, or even 300, so that the description can be a couple of sentences.

Reply:
Sadly, [Google limits to 160 characters](https://moz.com/learn/seo/meta-description), but I agree a longer limit is probably better.